images/demo/Dockerfile.jupyter

```diff
@@ -46,34 +46,30 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 
 
 # Kamu
-RUN wget -q https://github.com/kamu-data/kamu-cli/releases/download/v$KAMU_VERSION/kamu-cli-x86_64-unknown-linux-gnu.tar.gz && \
-    tar -xf kamu-cli-x86_64-unknown-linux-gnu.tar.gz && \
-    chmod +x kamu-cli-x86_64-unknown-linux-gnu/kamu && \
-    mv kamu-cli-x86_64-unknown-linux-gnu/kamu /usr/local/bin/ && \
-    rm -rf kamu-cli-x86_64-unknown-linux-gnu* && \
-    echo "source <(kamu completions bash)" >> /home/$NB_USER/.bashrc
-COPY kamu/.kamuconfig /.kamuconfig
+RUN curl -s "https://get.kamu.dev" | KAMU_VERSION=$KAMU_VERSION KAMU_INSTALL_PATH=/usr/local/bin/kamu sh && \
+    echo "source <(kamu completions bash)" >> /home/$NB_USER/.profile
+COPY jupyter/.kamuconfig /.kamuconfig
 
+RUN chown -R $NB_USER:users /home/$NB_USER
 
 #########################################################################################
 USER $NB_USER
 
 # This is needed because requests-kerberos fails to install on debian due to missing linux headers
 RUN conda install requests-kerberos==0.12.0 -y
 
-COPY kamu/requirements.txt requirements.txt
+COPY jupyter/requirements/$TARGETPLATFORM/requirements.txt requirements.txt
```

Comment:
Note that using `$TARGETPLATFORM` is the only way to branch execution between platforms. Since the values can be `linux/amd64, linux/arm64` (include slash) I had to match the directory layout of the requirements files.